web/test/data/projects/project-public-full.json

```diff
@@ -0,0 +1,24 @@
+{
+  "uid": "test-project-project-monitors",
+  "type": "public",
+  "title": "Test public project with full data title",
+  "description": "Test public project with full data description",
+  "url": "https://www.pipelinedashboard.io",
+  "repositories": ["test-repository-minimum"],
+  "monitors": [{
+    "uid": "test-monitor",
+    "name": "test monitor",
+    "method": "GET",
+    "path": "/",
+    "expectedCode": 200,
+    "expectedText": "",
+    "successfulPings": 0,
+    "unsuccessfulPings": 0
+  }],
+  "pings": [],
+  "tokens": [],
+  "views": 1,
+  "access": {
+    "admin": [ "00000000-0000-0000-0000-000000000001" ]
```

Comment:
`uid`s format is like a namespace `test-{model}-{type}-{uniqueness}`, this was probably from my commit, I will update

Reply:
ok